styles.css

```diff
@@ -0,0 +1,920 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+:root {
+  --clr-purple: #6070ff;
+  --clr-white: #fff;
+  --clr-para: #344563;
+  --clr-gray-100: #b3bac5;
+}
+
+* {
+  margin: 0;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  background-color: #e5e5e5;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.logo a {
+  color: var(--clr-purple);
+  font-weight: 700;
+  font-size: 18px;
+}
+
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+  background-color: var(--clr-white);
+  position: fixed;
+  left: 0;
+  right: 0;
+  z-index: 1;
+  align-items: center;
+}
+
+.dev a {
+  font-weight: 500;
+  color: #7f8cff;
+}
+
+.nav-link a {
+  color: #fff;
+}
+
+.social-icons a {
+  margin-right: 1.2rem;
+}
+
+.nav-link a:hover {
+  color: var(--clr-para);
+}
+
+.menu-icon > span {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: var(--clr-purple);
+}
+
+.menu-icon.active span {
+  transform: translateY(8px) rotate(45deg);
+}
+
+/* .menu-icon.active span {
+  transform: translateY(-8px) rotate(-45deg);
+} */
```

Comment:
Great job implementing the changes requested by the previous reviewer 💯. However, there is still a small issue with the X menu button.

This is how your menu currently looks like while open: 

![xerror](https://user-images.githubusercontent.com/111214189/225985813-770beff0-1cc8-492a-baac-45f9ad566597.png)

And this is what the Figma template looks like:

![xtempl](https://user-images.githubusercontent.com/111214189/225986171-c7b4c656-959a-424a-86fe-f6d2ca8f6ed2.png)

You can easily fix this issue by using the correct classes for your button in your CSS. For example, you can change `.menu-icon.active span` for `.menu-icon.active .bar1`. You can also uncomment lines 74-76 and do the same but with `.bar3`.

This way, your project is going to look almost identical to the Figame template, which is one of the requirements. It will also help your portfolio look more professional 🤝 .